lessons/basics/composition.md

```diff
@@ -21,11 +21,11 @@ Modules are the best way to organize functions into a namespace.  In addition to
 Let's look at a basic example:
 
 ``` elixir
-defmodule Example do
-  def greeting(name) do
-    ~s(Hello #{name}.)
-  end
-end
+iex> defmodule Example do
```

Comment:
I'm not sure about these changes.  I intentionally leave off the iex prompt on larger blocks so they can be copied and pasted. 

@3100 or @erickgnavar do either of you have any thoughts?